mycroft/client/speech/main.py

```diff
@@ -99,6 +99,14 @@ def handle_mic_unmute(event):
     loop.unmute()
 
 
+def handle_mic_is_muted(event):
+    """
+        Query microphone mute status.
+    """
+    data = {'mic_state': loop.is_muted()}
+    ws.emit(Message('mycroft.mic.muted', data))
```

Comment:
I suggest we start using the .response method on the Message object. It hasn't been widely used yet but is basically intended for this. In this case it would become:

```python
ws.emit(event.response(data)
```

would generate a new message from the original message type adding `.response` so the response message would become `mycroft.mic.is_muted.response` and is automatically handled when using the

`ws.wait_for_response`

Reply:
Thanks for the review, made the changes to the commit, on testing response is now mycroft.mic.is_muted.response data {mic_state: true/false}